Dockerfile

```
@@ -1,4 +1,4 @@
-ARG PYTHON_VERSION=2.7
+ARG PYTHON_VERSION=2.7-slim
 
 FROM python:${PYTHON_VERSION}
```

I think including it in the `ARG` could break some uses (e.g., if CI specifies the python version);

Could you instead add it here?

```suggestion
FROM python:${PYTHON_VERSION}-slim
```

Looks like the python-3 version of the Dockerfile also needs the same change; https://github.com/docker/docker-py/blob/master/Dockerfile-py3#L3

(also wondering if we should remove the Python 2 variant, as it reached EOL